package.json

```diff
@@ -100,6 +100,7 @@
     "analyze": "source-map-explorer 'build/static/js/*.js'",
     "start": "node ./server",
     "dev": "concurrently 'nodemon ./server' 'craco start' 'cd src/services/SeedInfo/noita_random && while inotifywait -e close_write **; do sh build.sh; done;'",
+    "dev-win": "concurrently \"nodemon ./server\" \"craco start\"",
```

Comment:
I added it. Because there is nothing like `inotifywait` on Windows, I did not re-add it back to the dev-win script. Instead, I added a separate script that builds the wasm files and in addition, create a VS Code task to build the files on demand within the IDE.